app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
```

Comment:
Assigning `Person.people[name] = self` without checking for existing entries can lead to overwriting existing instances if duplicate names are provided. Consider checking for duplicates or using unique identifiers.